aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -1379,6 +1427,34 @@ module aptos_token::token {
         assert!(property_map::length(&token.token_properties) == 2, property_map::length(&token.token_properties));
     }
 
+    #[test(creator = @0xAF, owner = @0xBB)]
+    fun test_mutate_token_uri(creator: &signer, owner: &signer) acquires Collections, TokenStore {
```

Comment:
Can you also add a test where mutating token uri fails because it's not allowed?